Source/Orts.Formats.OR/WeatherFile.cs

```diff
@@ -101,15 +102,18 @@ public WeatherSettingOvercast(JsonReader json)
 
         protected override bool TryParse(JsonReader item)
         {
+
+            // get values
             if (base.TryParse(item)) return true;
             switch (item.Path)
             {
-                case "Overcast": Overcast = item.AsFloat(Overcast); break;
-                case "OvercastVariation": OvercastVariation = item.AsFloat(OvercastVariation); break;
-                case "OvercastRateOfChange": OvercastRateOfChange = item.AsFloat(OvercastRateOfChange); break;
+                case "Overcast": Overcast = item.AsFloat(Overcast) >= 0 ? item.AsFloat(Overcast) : -1; break;
```

Comment:
What's the purpose of this ternary? Why would you force all negative values to be -1?

Reply:
All checks on negative values have been removed.